packages/bpk-component-breadcrumb/src/BpkBreadcrumbItem.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { cssModules } from 'bpk-react-utils';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import BpkButton from 'bpk-component-button';
+import Chevron from 'bpk-component-icon/sm/arrow-right';
+import { withRtlSupport } from 'bpk-component-icon';
+
+import STYLES from './BpkBreadcrumbItem.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkBreadcrumbItem = props => {
+  const { children, active, href, ...rest } = props;
+
+  const classNames = getClassName(
+    'bpk-breadcrumb-item',
+    active && 'bpk-breadcrumb-item--active',
+  );
+  const RtlSupportedChevron = withRtlSupport(Chevron);
```

Comment:
Can we move this outside this function? We don't need to recreate the HOC in every render.